test/desktop-lite/scenarios.json

```diff
@@ -0,0 +1,18 @@
+{
+    "desktop-lite_noble_numbat": {
+        "image": "ubuntu:noble",
```

Comment:
Thanks for adding these tests. However, we already have workflows which are automatically adding these tests. 

Can we remove the scenarios? Thanks!

Reply:
done 👍